script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt

```diff
@@ -1,107 +1,119 @@
 }
 [sha:ba682b3532e5ce71fd05ceb771c85d1ef5f382ce]{
 -- Page 35, Offset 39441.
 -- 「落ちる……！」
-"It's falling...!"
+"It's falling!"
 }
 [sha:2b54f83ae3177704b421602cef615f5691032337]{
 -- Page 36, Offset 39442.
 -- 「………終わった、のか？」
-"...Is it over?"
+"Is it over...?"
 }
 [sha:f7306b13383b2d247d3174460caac3aa5f26603e]{
 -- Page 36, Offset 39443.
 -- 街は静寂に包まれた。
-The city is wrapped in silence.
+Silence descends over the city.
 }
 [sha:4dbc99f5d64d186a230e30e500459aabdbe1c08b]{
 -- Page 36, Offset 39444.
 -- 先ほどまでの閃光は見る影もなく、銃声も、ビルの崩れる音も途絶えた。
-There isn't any trace of the flash of light from earlier, and the sound of gunshots and crumbling buildings comes to an end.
+// I feel like it stuck around a little too long to be considered a 'flash'
+The blinding light that was there a second ago is gone,
+ and I can't hear the sound of gunfire or of buildings being destroyed.
 }
 [sha:b75c906778e5901eec12ad2a39644b00272b1dcf]{
 -- Page 37, Offset 39445.
 -- 「間違いない。シエル先輩の勝ちだ……！」
-"There's no doubt about it. Ciel is the victor...!"
+"There's no doubt about it. Ciel-senpai is the victor!"
 }
 [sha:ff7ed4707ca3c856fabbdc5881378091b8662894]{
 -- Page 37, Offset 39446.
 -- スポーツ観戦じゃあるまいし、事の勝ち負けに一喜一憂している場合じゃないのは承知している。
-I'm fully aware that this isn't a sports match, and that this isn't the time to rejoice over winning or losing.
+// Kinda tenuous maybe for 勝ち負けに一喜一憂, but I interpret it as like
+// "vicariously suffering the joy/sorrow of the wins/losses of your team"?
+I realize that this isn't a sports match,
+ and that I shouldn't be getting too caught up in rooting for my team.
 }
 [sha:d88c59592f71d6379825c3897b88623dff422f3a]{
 -- Page 37, Offset 39447.
 -- それでも先輩……シエルが無事であってくれて嬉しい。
-But even so, I'm happy that Senpai... Ciel was able to make it out in peace.
+But even so, I'm happy that Senpai――that Ciel was able to make it out in one piece.
 }
 [sha:129de27b6a3299432f37a5f069154030c64b8224]{
 -- Page 37, Offset 39448.
 -- 彼女の奮戦が酬われた事が、我がことのように誇らしい。
-I'm proud of her being rewarded for her hard work as if it were my own.
+I feel a secondhand pride at the thought that her perseverance through this hard-fought battle has paid off.
 }
 [sha:0b860f691ded0cb09b6fc264e9729ac02a0b11b3]{
 -- Page 38, Offset 39449.
 -- だが。その喜びもすぐに終わってしまった。
-However, that feeling of joy came to an end in no time.
+However――that feeling of joy does not last.
 }
 [sha:b473ee52e6fd2ed63a6c9c03296a43dae93eabf0]{
 -- Page 38, Offset 39450.
 -- 「ぁ……く、ぅ……！」
-"Ah... kh, gh...!"
+"%{i}Ah%{/i}... %{i}kh%{/i}, %{i}ghh%{/i}!"
 }
 [sha:39add63def11e80c0783495535a4217673287e09]{
 -- Page 38, Offset 39451.
 -- 足がふらついて倒れそうになる。
-My legs stagger and comes close to collapse.
+My legs turn to jelly, and I almost collapse.
 }
 [sha:53eb5cbad6b30593ea48dc93c1b04131f4ef511a]{
 -- Page 38, Offset 39452.
 -- 理由のない胸焼けに襲われて、吐きそうになった。
-I'm struck by an unknown feeling in my stomach, and I feel like I'm about to throw up.
+I develop heartburn out of nowhere,
+ and feel like I'm about to throw up.
```

Comment:
Prefer current

Reply:
"develop heartburn out of nowhere" seems like a pretty weird way of putting it. It's as if he's gotten a new disease, rather than it being a momentary feeling.
That said, I think inexplicable is preferrable. Maybe the verb can be swapped too. Also the I needed to change place, I forgot that.
>An inexplicable heartburn befalls me, and I feel like...